docs/content.zh/release-notes/flink-1.19.md

```diff
@@ -0,0 +1,362 @@
+called automatically before the execution of any processing methods(map, join, etc.). If the new
+`open(OpenContext openContext)` method is not implemented, Flink will fall back to invoking the
+deprecated `open(Configuration parameters)` method.
+
+#### Deprecate API that uses Flink's Time implementation (related to FLINK-14638)
+
+##### [FLINK-32570](https://issues.apache.org/jira/browse/FLINK-32570)
+
+Flink's Time classes are deprecated now and will be subject to deletion with the release of Flink 2.0.
+Please start to use Java's own `Duration` class, instead. Methods supporting the `Duration` class
+that replace the deprecated Time-based methods were introduced.
+
+#### Add new interfaces for SinkV2 to synchronize the API with the Source API
+
+##### [FLINK-33973](https://issues.apache.org/jira/browse/FLINK-33973)
+
+According to [FLIP-372](https://cwiki.apache.org/confluence/display/FLINK/FLIP-372%3A+Enhance+and+synchronize+Sink+API+to+match+the+Source+API) the SinkV2 API has been changed.
+The following interfaces are deprecated: `TwoPhaseCommittingSink`, `StatefulSink`, `WithPreWriteTopology`, `WithPreCommitTopology`, `WithPostCommitTopology`.
+The following new interfaces have been introduced: `CommitterInitContext`, `CommittingSinkWriter`, `WriterInitContext`, `StatefulSinkWriter`.
+The following interface method's parameter has been changed: `Sink#createWriter`
+The original interfaces will remain available during the 1.19 release line, but they will be removed
+in consecutive releases. For the changes required when migrating, please consult the Migration Plan detailed in the FLIP.
+
+
+#### Deprecate configuration getters/setters that return/set complex Java objects
+
+##### [FLINK-33581](https://issues.apache.org/jira/browse/FLINK-33581)
+
+The non-ConfigOption objects in the `StreamExecutionEnvironment`, `CheckpointConfig`, and 
+`ExecutionConfig` and their corresponding getter/setter interfaces are now be deprecated in [FLINK-33581](https://issues.apache.org/jira/browse/FLINK-33581).
+And these objects and methods are planned to be removed in Flink 2.0. The deprecated interfaces
+include the getter and setter methods of `RestartStrategy`, `CheckpointStorage`, and `StateBackend`.
+More details can be found at [FLIP-381](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=278464992).
+
+
+### Table SQL / API
+
+#### Support Setting Parallelism for Table/SQL Sources
+
+##### [FLINK-33261](https://issues.apache.org/jira/browse/FLINK-33261)
+
+Scan table sources can now be set a custom parallelism for performance tuning via the `scan.parallelism`
+option. Currently, only the DataGen connector has been adapted to support that, Kafka connector is
+on the way. Please check [scan-table-source](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sourcessinks/#scan-table-source) on how to adapt your custom connectors to it.
```

Comment:
I think we should align with the announcement blog to change the doc link from `flink-docs-master` to `flink-docs-release-1.19`.